### ISSUE TITLE
Windows Zip -> Trojan:Win64/CobaltStrike.HP!MTB

### ISSUE THREAD

**Issue by Maxiviper117** [h3rald/litestore#74]
Microsoft flags zip for windows as virus:

![image](https://github.com/h3rald/litestore/assets/11472492/8813f2fc-be1d-47a9-a2d1-b64a7011d766)

**Comment by h3rald** [comment]
Yep, I had the same problem when I downloaded it on my work laptop. Really annoying actually, especially because all litestore executables are built using GitHub actions, and I also *especially* built OpenSSL using another set of automated GitHub actions (see [this project](https://github.com/h3rald/openssl-build/tree/master)).

In my case, I recompiled on my windows machine and the generated executable didn't trigger anything.

I don't know how to fix this, suggestions are welcome. And no, I really can't be bothered to spend money to get a certificate to sign the windows executables...
